scripts/export/dump_phenotypes_wormbase.pl

```diff
@@ -0,0 +1,193 @@
+my $registry = 'Bio::EnsEMBL::Registry';
+$registry->load_registry_from_db(
+    -host       => $host,
+    -port       => $port,
+    -user       => $user,
+    -pass       => '',
+    -db_version => $e_version,
+ );
+
+my %only_species = map { $_ => 1 } split(",",$species) if defined $species;
+
+my $sql = qq{SHOW DATABASES LIKE '%variation_${p_version}_${e_version}%'};
+my $sth_h = get_connection_and_query($database, $hname, $sql, 1);
+
+my $sqlPF = qq{SELECT count(*) FROM phenotype_feature};
+
+# Loop over databases
+while (my ($dbname) = $sth_h->fetchrow_array) {
+
+  next if ($dbname =~ /^master_schema/ || $dbname =~ /private/);
+
+  next if ($dbname !~ /^[a-z]+_[a-z]+_[a-z0-9]+_variation_\d+_\d+_\d+$/i);
+
+  my $cdbname = `sed 's/variation/core/g' <<< $dbname`;
+  #print "Core: ".$cdbname."\n";
+
+  my $ftpfilename = ProductionMysql::core_db_to_local_ftp_filename(ProductionMysql->staging, $cdbname);
+
+  $dbname =~ /^(.+)_variation_.+_(.+)/;
+  my $s_name = $1;
+  my $assembly = $2;
+
+  next if defined $species & !$only_species{$s_name};
+  print STDERR "# $s_name - [ $dbname ]\n";
+
+  ## count phenotype_feature(s)
+  my $sth_pf = get_connection_and_query($dbname, $hname, $sqlPF, 1);
+  my ($count) = $sth_pf->fetchrow_array();
+  print "Found phenotype features: ", $count, "\n";
+  next unless $count > 0;
+
+  my $dumpFile = sprintf($output_dir."/".$ftpfilename."."."phenotypes.wb", $s_name, $e_version);
+
+  die ("Phenotype file $dumpFile already exists. Specify a different output folder or use --force to override the existing one\n") if -e $dumpFile & !$force ;
+
+  print "writing: $dumpFile \n" unless $quiet;
+
+  my %reg_config = (reg => $registry, species => $s_name, quiet => $quiet, dbname => $dbname, cdbname => $cdbname);
+  my %db_config = (config => \%reg_config);
+
+  #Run Phenotypes.pm plugin to export GVF formated Phenotypes
+  Phenotypes::generate_phenotype_gff(\%db_config, $dumpFile);
+}
+$sth_h->finish;
+
+
+
+# Connects and execute a query
+sub get_connection_and_query {
+  my $dbname  = shift;
+  my $hname   = shift;
+  my $sql     = shift;
+  my $rtn_sth = shift;
+
+  my ($host, $port) = split /\:/, $hname;
+
+  # DBI connection 
+  my $dsn = "DBI:mysql:$dbname:$host:$port";
+  my $dbh = DBI->connect($dsn, $user) or die "Connection failed";
+
+  my $sth = $dbh->prepare($sql);
+  $sth->execute;
+
+  if ($rtn_sth) {
+    return $sth;
+  }
+  else {
+    $sth->finish;
+  }
+}
+
+
+sub usage {
+
+  print qq{
+  Usage: perl dump_phenotypes.pl [OPTION]
+
+  Dumping to GVF of phenotype data for Ensembl Variation species.
+
+  Options:
+
+    -help             Print this message
+
+    -output_dir|o     Output directory for the GVF dumps (Required)
+    -force|f          Force to override any pre-existing dumps
+    -quiet|q          Suppress warning messages.Not used by default
+
+    -species|s        Only export the species in the list,
+                        e.g. homo_sapiens,sus_scrofa
```

Comment:
Could you update the examples? Human and pig won't work with this script because of this line:
`next if ($dbname !~ /^[a-z]+_[a-z]+_[a-z0-9]+_variation_\d+_\d+_\d+$/i);`

Reply:
Hi @dglemos , that's why I created a new file (dump_phenotypes_wormbase.pl) instead of using the general one.

Reply:
Hi @dglemos, you're right I get your point! 
I fixed it, could you please have a look? :)